packages/core/src/Stores/notification-store.js

```diff
@@ -191,7 +191,7 @@ export default class NotificationStore extends BaseStore {
         const virtual_account = landing_company_shortcode === 'virtual';
         const is_website_up = website_status.site_status === 'up';
         const has_trustpilot = LocalStore.getObject('notification_messages')[loginid]?.includes(
-            this.client_notifications.trustpilot.key
+            this.client_notifications?.trustpilot?.key
```

Comment:
`client_notifications` and `trustpilot` should not be undefined because `setClientNotifications` is called before 
`handleClientNotifications` is called. Can you please find out the root cause?

Reply:
@balakrishna-binary @yashim-deriv  The root issue was `setClientNotifications` was not called before `handleClientNotifications` in refreshNotifications function. And it is now fixed by this PR https://github.com/binary-com/deriv-app/pull/5199

` refreshNotifications() {
        this.removeNotifications(true);
        this.removeAllNotificationMessages();
        **this.setClientNotifications();**
        this.handleClientNotifications();
    }`